pyaerocom/io/cams2_83/reader.py

```diff
@@ -443,6 +458,11 @@ def read_var(self, var_name: str, ts_type: str | None = None, **kwargs) -> Gridd
     data_id = "CAMS2-83.EMEP.day0.AN"
     reader = ReadCAMS2_83(data_dir=data_dir, data_id=data_id)
     reader.daterange = ("2021-12-01", "2021-12-04")
+    print(
+        np.unique(reader.daterange.values.astype("datetime64[Y]").astype("int") + 1970).astype(
+            "str"
+        )
+    )
     print(reader.filepaths)
     # dates = ("2021-12-01", "2021-12-04")
```

Comment:
Dead code?

Reply:
This is example code in the main/test-section which should be kept.